src/source/reference/get-feed-import-id-detail.rst

```diff
@@ -1,8 +1,91 @@
 .. index:: GET /feed/import/{id}/detail
+
 .. _get_feed_import_id_detail:
 
-GET /feed/import/{id}/detail
-============================
+.. _get_feed_import_id_detail_v2:
+
+GET /feed/import/{id}/detail V2
```

Comment:
please check how the rest of the documentation has separate parts for V1 and V2, where the header (GET /feed/import/{id}/detail) is there only once. For instance, check GET /ad/{id}

Reply:
the detail header is only once, but there is also a _get_feed_import_id_detail_v2 which is the same as for /ad/{id} from what I can see, it's just moved under the title - I did the same now